.github/workflows/build-and-test.yml

```diff
@@ -25,7 +25,7 @@ jobs:
         run: git fetch --depth=100 origin +refs/tags/*:refs/tags/*
 
       - name: Set up JDK 11
-        uses: coursier/setup-action@v1
+        uses: coursier/setup-action@v1.2.0-M2
```

Comment:
@ihostage, I will change this to 1.1.2 as you suggested, but I want to see a green build first. :-)

Reply:
Of course 😄 
I just remembered that when I saw your changes because I was checking the `v1` tag for this action and found that it was outdated. 😞 
I wish I don't remember early on the first review 🤦‍♂️